### PR TITLE
Added version 0.32 to apt-get trimmomatic removed trimmomatic.sh file

### PR DESCRIPTION
**Pull request by alexbarrera**

### CODE REVIEW

File: trimmomatic/Dockerfile [hunk 1]
```diff
@@ -3,10 +3,10 @@ MAINTAINER Dan Leehr <dan.leehr@duke.edu>
...
+  trimmomatic="0.32+dfsg-1"
 
 # Copy wrapper script
 
-COPY trimmomatic.sh /usr/bin/trimmomatic.sh
+ENV PATH=/usr/share/java/:$PATH
 
-CMD ["/usr/bin/trimmomatic.sh"]
\ No newline at end of file
+CMD ["trimmomatic.jar"]
```
Comment (dleehr) [inline trimmomatic/Dockerfile]: The `.jar` file is not executable, so setting it as `CMD` results in the following when I try to run the container:

    $ docker run dukegcb/trimmomatic
    exec: "trimmomatic.jar": executable file not found in $PATH
    Error response from daemon: Cannot start container e2225de46534b5021a1e5b2d3fe93129fb1214619251b340e513f8146addec0d: [8] System error: exec: "trimmomatic.jar": executable file not found in $PATH

This should probably be something like

    ["/usr/bin/java", "-jar", "/usr/share/java/trimmomatic.jar"]

Making the `PATH` change unnecessary

File: trimmomatic/Dockerfile [hunk 2]
```diff
@@ -3,10 +3,10 @@ MAINTAINER Dan Leehr <dan.leehr@duke.edu>
 
 RUN apt-get update && apt-get install -y \
   openjdk-7-jre-headless \
-  trimmomatic
+  trimmomatic="0.32+dfsg-1"
 
 # Copy wrapper script
```
Comment (dleehr) [inline trimmomatic/Dockerfile]: Comment should be removed if the wrapper script is not copied

**Comment by dleehr** [comment]
Thanks, this is great. A couple comments though

**Comment by alexbarrera** [comment]
You are right. I fixed both comments.

**Comment by dleehr** [comment]
:+1:

**Pull request merged**